plugin/src/lib.rs

```diff
@@ -319,9 +319,9 @@ impl<'a, 'cx> Folder for MutatorPlugin<'a, 'cx> {
                             ],
                         );
                     }
-                    let left = left.map(|e| fold::noop_fold_expr(e, self));
-                    let right = right.map(|e| fold::noop_fold_expr(e, self));
-                    quote_expr!(self.cx, ::mutagen::and(|| $left, || $right, $n))
```

Comment:
Am I correct when I say that you should not remove the `::` in the last line? I think that was actually added in another commit recently